.github/workflows/integration-flows.yaml

```diff
@@ -37,12 +37,7 @@ jobs:
                       mkdir -p nango-integrations
                       find . -maxdepth 1 ! -name '.' ! -name 'nango-integrations' -exec mv -t nango-integrations {} +
                       cd nango-integrations
-                      output=$(nango compile 2>&1)
-                      echo "$output"
-                      if [[ "$output" == *"Error compiling"* ]]; then
-                        echo "Error found in directory $dir"
-                        exit 1
-                      fi
+                      NANGO_CLI_UPGRADE_MODE=ignore nango compile
```

Comment:
I don't think we need to look at the output. Any failure to compile should be reported and fails the action. What do you think @khaliqgant

Reply:
That sounds right in theory but I seem to remember needing the output because it silently fails. 

We can see if it works and add it back in if not I suppose

Reply:
It is not that it fails silently, the problem is that ` nango compile` exitCode is 0 even when some compilation fails. I'll check if this can be fixed.

Reply:
Ah right. Think that is intentional so one failing script doesn’t block others from compiling and even deploying

Reply:
It is great that `compileService.run()` doesn't stop when encountering an error but the CLI command should fails if success is not true. what do you think? I am thinking about this kind of behavior: https://github.com/NangoHQ/nango/commit/b3b4d88e99bc68ebe43b1ebfe244bc9428be3656

Reply:
Can we add test cases for the behavior that is desired?

https://github.com/NangoHQ/nango/blob/master/packages/cli/lib/sync.unit.test.ts